pkg/webhooks/regularuser/regularuser.go

```diff
@@ -307,6 +322,25 @@ func isNetNamespaceValid(s *RegularuserWebhook, request admissionctl.Request) bo
 	return true
 }
 
+// allow if a ConfigMap is being updated that does not live under openshift-config or is not called user-ca-bundle under openshift-config
+func shouldAllowConfigMapChange(s *RegularuserWebhook, request admissionctl.Request) bool {
+	decoder, err := admissionctl.NewDecoder(&s.s)
+	if err != nil {
+		return false
+	}
+	configMap := &corev1.ConfigMap{}
+	err = decoder.DecodeRaw(request.Object, configMap)
+	if err != nil {
+		return false
+	}
+
+	if configMap.ObjectMeta.Name != "user-ca-bundle" || configMap.ObjectMeta.Namespace != "openshift-config" {
```

Comment:
Is it flexible to make "user-ca-bundle" and "openshift-config" as a var list like `Resources:   []string{"clusterversions", "clusterversions/status", "schedulers", "apiservers", "proxies"},` above?

Reply:
other functions also doing the similar with hard-coded strings. I feel ok about this.